configuration-api/main/config.js

```diff
@@ -1,4 +1,5 @@
 let logger = require('logops');
+const dotenv = require('dotenv');
```

Comment:
testing locally with debug on, it was clear that when doing `source .env` env variable where not actually loaded on the mac.